server/src/test/java/com/lunchtime/controllers/CategoryFoodControllerTest.java

```diff
@@ -21,10 +23,14 @@
     private CategoryFoodService mockCategoryFoodService;
     private CategoryFoodController categoryFoodControllerUnderTest;
 
+    @Mock
+    CategoryFoodController mockCategoryFoodController;
+
     @Before
     public void setUp() {
         initMocks(this);
         categoryFoodControllerUnderTest = new CategoryFoodController(mockCategoryFoodService);
+        mockCategoryFoodController = new CategoryFoodController(mockCategoryFoodService);
```

Comment:
Why you add this mock, but don't use it in tests?

Reply:
Fixed it.